### PR TITLE
Forms 2980 email tel default validaiton

### PR DESCRIPTION
**Pull request by egmox**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

**Comment by egmox** [comment]
the default validation pattern is no pattern for now which is coming from the cq-guides

image

File: bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FieldType.java [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
 public enum FieldType {
     TEXT_INPUT("text-input"),
+    EMAIL_INPUT("email-input"),
```
Comment (rismehta) [inline bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FieldType.java:24]: FieldType should be email, why is this email-input ?

**Comment by rismehta** [comment]
Please add a runtime test case for the validation pattern in cypress

**Review by rismehta — CHANGES_REQUESTED** [review]
-1 for now

**Pull request closed without merging**